pkg/apiserver/common/common.go

```diff
@@ -44,16 +44,19 @@ type Pod struct {
 type Service struct {
 	conf    *config.ChaosDashboardConfig
 	kubeCli client.Client
+	Reader  client.Reader
```

Comment:
Is it necessary to make this field public? Why not just consistent as other fields